lib/genre.rb

```diff
@@ -0,0 +1,34 @@
+class Genre
+  extend Concerns::Findable
+
+  attr_accessor :name
+  attr_accessor :songs
+
+  @@all = []
+
+  def initialize(name)
+    @name = name
+    @songs = []
+  end
+
+  def self.all
+    @@all
+  end
+
+  def save
+    @@all << self
+    self
+  end
+
+  def self.destroy_all
+    @@all.clear
+  end
+
+  def self.create(name)
+    Genre.new(name).save
+  end
+
```

Comment:
#### Code Quality - Critical
```self.all``` ```save``` ```self.destroy_all``` and ```self.create``` can be refactored into a parent class. This way they are not repeated in the ```Song``` and ```Artist``` classes.